nmi_mysql/nmi_mysql.py

```diff
@@ -3,22 +3,67 @@
     Useful for raw queries and scripting
 """
 
+import re
+import inspect
+import logging
+import warnings
+
+from functools import wraps
+
 from pymysql.cursors import DictCursor
 
 from sqlalchemy import create_engine
 from sqlalchemy.exc import SQLAlchemyError
 
-import re
-import logging
-
-
 CONFIG_KEYS = ['host', 'user', 'password', 'db', 'port']
 MAX_POOL_SIZE = 10
 
 
-class DB(object):
+# NOTE(kgriffs): We don't want our deprecations to be ignored by default,
+# so create our own type.
+class DeprecatedWarning(UserWarning):
+    pass
+
+
+# https://gist.githubusercontent.com/kgriffs/8202106/raw/e2f7b64d732e598679ae12954a993f7384130158/util.py
+def deprecated(instructions):
```

Comment:
Remove this function in this file.
Maybe create a separate file or util file that will host all the functions like this.